plugins/pulp_rpm/plugins/distributors/yum/configuration.py

```diff
@@ -517,3 +520,24 @@ def _check_for_relative_path_conflicts(repo, config, config_conduit, error_messa
         error_messages.append(msg.format(relative_path=relative_path, repo_id=repo.repo_id,
                                          conflict_url=conflicting_relative_url,
                                          conflict_repo=conflicting_repo_id))
+
+
+def _check_repoview_sqlite_conflicts(config, error_messages):
```

Comment:
A couple of simple unit tests would be a natural fit for this. At least one where ``repoview`` is ``True``, and one where it's ``False``.